### PR TITLE
Modify doc and swap-cli

### LINKED ISSUE #216: commands to remove from swap-cli
connect
ping
listen

### LINKED ISSUE #262: readme / docker needs to be updated to reflect new config structure

### PR DESCRIPTION
**Pull request by h4sh3d**
Fix #262, fix #216

Change swap-cli make|take command to use named arguments an allow better usage of default values (some have been updated). Remove default values for exchanged amounts.

Update overall documentation for docker with new config and new command line arguments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,25 +129,22 @@ When listening for other peers to connect, e.g. for executing a swap, a `peerd`
...
 ```
-swap-cli make tb1q935eq5fl2a3ajpqp0e3d7z36g7vctcgv05f5lf\
-    54EYTy2HYFcAXwAbFQ3HmAis8JLNmxRdTC9DwQL7sGJd4CAUYimPxuQHYkMNg1EELNP85YqFwqraLd4ovz6UeeekFLoCKiu\
-    Testnet ECDSA Monero\
-    "0.00001350 BTC" "0.001 XMR"\
-    Alice 4 5 "1 satoshi/vByte"\
-    1.2.3.4\
-    0.0.0.0\
-    9735
+swap-cli make --arb-addr tb1q935eq5fl2a3ajpqp0e3d7z36g7vctcgv05f5lf\
```
Comment (TheCharlatan) [inline README.md:132]: I'd prefer an example with all flags either here, or in local-swap, even if they are just passing default values.

Comment (h4sh3d) [reply README.md:132]: Done in 4a9baf3d586117f0632e22a909491a09ac6a1da0

**Review by TheCharlatan — APPROVED** [review]
I'm not sure how I feel about getting rid of the positional arguments for the addresses, but we can work on ergonomics later.

File: README.md [hunk 2]
```diff
@@ -70,24 +76,43 @@ You can follow the documentation on [how to use `farcasterd` with Docker](./doc/
 
 Running `monero-wallet-rpc` is also possible with Docker, see [run `monero rpc wallet`](./doc/docker-stack.md#run-monero-rpc-wallet) for more details.
 
-### :bulb: Use public infrastructure
+### Configuration
+
+`farcasterd` can be configured through a toml file located by default at `~/.farcaster/farcasterd.toml`, if no file is found `farcasterd` is launched with some default values. You can see an example [here](./farcasterd.toml).
```
Comment (zkao) [inline README.md:81]: platform specific?

Comment (zkao) [reply README.md:81]: `~/.farcaster/farcasterd.toml`

**Review by zkao — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (h4sh3d) [reply README.md:81]: You right, I'll generalized the doc to talk about others location of the default config file.

Comment (h4sh3d) [reply README.md]: Done in b527610b4dd86ae60af08f61b25d2c964f57cf43, is that better?

**Comment by h4sh3d** [comment]
Let me rebase on main now that more tests are available

**Review by TheCharlatan — APPROVED** [review]

**Review by TheCharlatan — APPROVED** [review]

**Pull request merged**